types/nodegit/object.d.ts

```diff
@@ -3,20 +3,35 @@ import { Repository } from './repository';
 import { Buf } from './buf';
 
 export class Object {
-    static size(type: number): number;
-    static lookup(repo: Repository, id: Oid, type: number): Promise<Object>;
-    static lookupPrefix(repo: Repository, id: Oid, len: number, type: number): Promise<Object>;
-    static string2type(str: string): number;
-    static type2string(type: number): string;
-    static typeisloose(type: number): number;
+    static size(type: Object.TYPE): number;
+    static lookup(repo: Repository, id: Oid, type: Object.TYPE): Promise<Object>;
+    static lookupPrefix(repo: Repository, id: Oid, len: number, type: Object.TYPE): Promise<Object>;
+    static string2Type(str: string): Object.TYPE;
```

Comment:
It appears that this should be `string2type` instead of `string2Type` [based on the docs](http://www.nodegit.org/api/object/#string2type)